service/jwks_fetching_roundtripper.go

```diff
@@ -0,0 +1,30 @@
+package service
+
+import "net/http"
+
+// ExternalErrorHandler is a type alias for a function which recieves an error and returns an error
+type ExternalErrorHandler func(error) error
```

Comment:
why the error return?

Reply:
Upon our agreement, I removed the return value.